src/frontend/src/lib/components/tokens/TokenCard.svelte

```diff
@@ -1,21 +1,31 @@
 <script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
 	import TokenLogo from '$lib/components/tokens/TokenLogo.svelte';
 	import TokenName from '$lib/components/tokens/TokenName.svelte';
 	import TokenSymbol from '$lib/components/tokens/TokenSymbol.svelte';
 	import Card from '$lib/components/ui/Card.svelte';
 	import { TOKEN_CARD, TOKEN_GROUP } from '$lib/constants/test-ids.constants';
+	import type { LogoSize } from '$lib/types/components';
 	import type { CardData } from '$lib/types/token-card';
 
 	export let data: CardData;
 	export let testIdPrefix: typeof TOKEN_CARD | typeof TOKEN_GROUP = TOKEN_CARD;
+	export let tokenCount: number | undefined = undefined;
```

Comment:
Should `tokenCount` actually becomes an optional variable or `CardData`?

Reply:
yes, makes sense, moved